### ISSUE TITLE
Seeking clarification on certain aspects of the article.

### ISSUE THREAD

**Issue by JACK-Chen-2019** [TaesikGong/NOTE#2]
Hello, I've gained a lot from reading your article, but there are some points that I don't quite understand.
1. How to obtain the result of Formula 4
2. Why update mean and var with Formula 5
Looking forward to your response. Thanks!
![image](https://github.com/TaesikGong/NOTE/assets/84917185/fbe1cee7-9fd8-48ca-8a57-4571ebbee152)

**Comment by TaesikGong** [comment]
Hi,
First of all, thank you for your interest in our work.

1.

I assume that you are aware of the term "sampling distribution" (https://en.wikipedia.org/wiki/Sampling_distribution) 
For the variance of the sample mean, please refer to "Standard error" section of this doc (https://en.wikipedia.org/wiki/Sampling_distribution).
![CleanShot 2023-08-16 at 18 37 10](https://github.com/TaesikGong/NOTE/assets/18747977/3273c33d-ef31-42dd-b469-954604b33785)
 You can obtain the same formula by squaring the written equation.

For the variance of the sample variance, please refer to "Distribution of the sample variance" section of this doc (https://en.wikipedia.org/wiki/Variance)
![CleanShot 2023-08-16 at 18 38 19](https://github.com/TaesikGong/NOTE/assets/18747977/67941f3d-7d1f-4e6c-8cef-3197bb0d2381)
You can see the same formula.


2.

Our idea is to correct normalization statistics only when the deviation between IN statistics and BN statistics is significant. The soft-shrinkage function (https://pytorch.org/docs/stable/generated/torch.nn.Softshrink.html) is suitable for this purpose.

I hope this helps, and please feel free to let us know if you have any more questions.
